spec/javascripts/packages/document-capture/components/document-capture-spec.jsx

```diff
@@ -297,6 +300,39 @@ describe('document-capture/components/document-capture', () => {
 
     const hasValueSelected = !!getByLabelText('doc_auth.headings.document_capture_front');
     expect(hasValueSelected).to.be.true();
+
+    // Submit button should be disabled until field errors are resolved.
+    submitButton = getByText('forms.buttons.submit.default');
+    expect(submitButton.classList.contains('btn-disabled')).to.be.true();
+    userEvent.upload(
+      getByLabelText('doc_auth.headings.document_capture_front'),
+      new window.File([''], 'upload.png', { type: 'image/png' }),
+    );
+    userEvent.upload(
+      getByLabelText('doc_auth.headings.document_capture_back'),
+      new window.File([''], 'upload.png', { type: 'image/png' }),
+    );
+
+    // Once fields are changed, their notices should be cleared. If all field-specific errors are
+    // addressed, submit should be enabled once more.
+    notices = await findAllByRole('alert');
+    const errorNotices = notices.filter((notice) => notice.classList.contains('usa-alert--error'));
+    expect(errorNotices).to.have.lengthOf(1);
+    expect(errorNotices[0].textContent).to.equal('An unknown error occurred');
+    expect(submitButton.classList.contains('btn-disabled')).to.be.false();
+
+    // Verify re-submission. It will fail again, but test can at least assure that the interstitial
+    // screen is shown once more.
+    userEvent.click(submitButton);
+    const interstitialHeading = getByText('doc_auth.headings.interstitial');
+    expect(interstitialHeading).to.be.ok();
+
+    await findAllByRole('alert');
+
+    expect(console).to.have.loggedError(/^Error: Uncaught/);
+    expect(console).to.have.loggedError(
+      /React will try to recreate this component tree from scratch using the error boundary you provided/,
+    );
```

Comment:
is this the way we catch that `findAllByRole('alert')` fails?

Reply:
It needs to be here only to avoid a race condition where the test finishes running before React has caught and rebuilt the application, because otherwise the error can still occur after the test has completed and cause failures with our [global console log spying](https://github.com/18F/identity-idp/blob/7c8f798f3cc7c27bba395dab0727dee50b839878/spec/javascripts/support/console.js#L55-L57).

Though, as I write this, maybe if we'd forcibly unmount everything in test teardown, we'd not encounter this...